modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -84,6 +85,10 @@ function acquia_cms_common_install($is_syncing) {
       'purge_ui',
     ]);
   }
+  else {
+    // Set acquia_cms_headless status variable.
```

Comment:
Change this message, instead of acquia_cms_headless use acquia_cms sync status may be.

Reply:
Done